run_pretrained_openfold.py

```diff
@@ -71,7 +71,10 @@ def precompute_alignments(tags, seqs, alignment_dir, args, is_multimer):
             fp.write(f">{tag}\n{seq}")
 
         if is_multimer:
-            local_alignment_dir = alignment_dir
+            local_alignment_dir = os.path.join(
```

Comment:
Ah yeah you're right, I've only run this with precomputed alignments. However, can you remove this conditional entirely and just make one statement since they are the same for both conditions

Reply:
Sure I've updated this part @christinaflo